test/performance/hamburg/build-config.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
@2martens Can you please add a Readme file in this folder with some contact information?